tests/vectors/inputs/publicKeyModel1.json

```diff
@@ -0,0 +1,13 @@
+{  
```

Comment:
from a types perspective, this is almost a linked data key pair, see: https://github.com/transmute-industries/did-key.js/blob/master/packages/test-vectors/src/did-core-conformance/p-256/p-256.json#L5

in the future, we might want to rename this model "SidetreeKeyPair" or something like that.